go.mod

```diff
@@ -6,27 +6,27 @@ require (
 	github.com/go-logr/logr v1.4.1
 	github.com/kr/text v0.2.0
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.8.1
-	github.com/onsi/ginkgo/v2 v2.14.0
-	github.com/onsi/gomega v1.30.0
-	github.com/operator-framework/operator-lib v0.12.0
-	github.com/operator-framework/operator-registry v1.35.0
-	github.com/prometheus/client_golang v1.18.0
-	github.com/prometheus/client_model v0.5.0
+	github.com/onsi/ginkgo/v2 v2.17.1
+	github.com/onsi/gomega v1.32.0
+	github.com/operator-framework/operator-lib v0.12.1-0.20240430173419-b394e34ce702
```

Comment:
@everettraven is it okay use a specific commit here? Or should we cut a new version of operator-lib?

Reply:
Should be okay to use a specific commit for now, but we do need to cut a release for operator-lib

Reply:
This could be `v0.13.0` since that is the version that has the 1.29 bump.